knowledge-base/kb-gridview-merged-cells-in-checkbox-column.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Display CheckBox in a Merged Cell for a Column Bound to a Boolean Property
```

Comment:
You can remove the "a"'s from here. It is allowed to omit this in the metadata (title, description, pagetitle).